packages/stark-ui/src/modules/dropdown/components/dropdown.component.scss

```diff
@@ -1,31 +1,26 @@
-/********* DROPDOWN *********/
-/* stark: src/modules/dropdown/components/dropdown.component.scss */
+/* ============================================================================== */
+/*  S t a r k   D r o p d o w n                                                   */
+/* ============================================================================== */
+/* stark-ui: src/modules/dropdown/components/dropdown/_dropdown.component.scss */
 .stark-dropdown {
   & mat-select {
```

Comment:
In fact the & can be removed here

Reply:
Thanks for the remark, I will adapt this.